policyengine_us/parameters/gov/contrib/congress/wftca/bonus_guaranteed_deduction/phase_out/rate.yaml

```diff
@@ -0,0 +1,9 @@
+description: Rate at which the bonus guaranteed deduction reduces with AGI.
+values:
+  2000-01-01: 0
```

Comment:
marginal_rate?

Reply:
Good point by since it's by filer status we can't I think